app/core/models.py

```diff
@@ -1,5 +1,150 @@
+
+    # user_status = models.ForeignKey(user_status_type, on_delete=models.PROTECT)
+    # current_practice_area = models.ManyToManyField("PracticeArea")
+    # target_practice_area = models.ManyToManyField("PracticeArea")
+
+    current_job_title = models.CharField(max_length=255, blank=True)
+    target_job_title = models.CharField(max_length=255, blank=True)
+    current_skills = models.CharField(max_length=255, blank=True)
+    target_skills = models.CharField(max_length=255, blank=True)
+
+    # desired_roles = models.ManyToManyField("Role")
+    # availability = models.IntegerField()  # not in ERD, is a separate table. Want to confirm to remove this
+    # referred_by = models.ForeignKey(referrer, on_delete=models.PROTECT) # FK to referrer
+
+    linkedin_account = models.CharField(max_length=255, blank=True)
+    github_handle = models.CharField(max_length=255, blank=True)
+    slack_id = models.CharField(max_length=11, blank=True)
+
+    phone = PhoneNumberField(blank=True)
+
+    texting_ok = models.BooleanField(default=True)
+
+    time_zone = TimeZoneField(blank=True, use_pytz=False, default="America/Los_Angeles")
+    # conduct = models.BooleanField()  # not in ERD. Maybe we should remove this
+
+    objects = UserManager()
+
+    USERNAME_FIELD = "username"
+    EMAIL_FIELD = "preferred_email"
+    REQUIRED_FIELDS = ["email"]  # used only on createsuperuser
+
+    @property
+    def is_django_user(self):
+        return self.has_usable_password()
+
+    def __str__(self):
+        return f"{self.email}"
+
+
+class Project(AbstractBaseModel):
+    """
+    List of projects
+    """
+
+    name = models.CharField(max_length=255, unique=True)
+    description = models.CharField(max_length=255, blank=True)
+    completed_at = models.DateTimeField("Completed at", null=True, blank=True)
+    github_org_id = models.CharField(
+        max_length=8,
+        blank=True,
+        help_text='Can be retrieved from gh api with the following: curl -H \
+"Authorization: token [gh_PAT]" https://api.github.com/orgs/[org]',
+    )
+    github_primary_repo_id = models.CharField(
+        max_length=9,
+        blank=True,
+        help_text='Can be retrieved from gh api with the following: curl -H \
+"Authorization: token [gh_PAT]" \
+https://api.github.com/repos/[org]/[repo]',
+    )
+    github_primary_url = models.CharField(max_length=255, blank=True)
+    # current_status_id = models.ForeignKey("status", on_delete=models.PROTECT)
+    hide = models.BooleanField(default=True)
+    # location_id = models.ForeignKey("location", on_delete=models.PROTECT)
+    slack_url = models.URLField(blank=True)
+    google_drive_url = models.URLField(blank=True)
+    google_drive_id = models.CharField(max_length=255, blank=True)
+    hfla_website_url = models.URLField(blank=True)
+    # leads = models.ManyToManyField("lead")
+    # leadership_type_id = models.ForeignKey("leadership_type", on_delete=models.PROTECT)
+    image_logo = models.URLField(blank=True)
```

Comment:
Should the image fields be [ImageFields](https://docs.djangoproject.com/en/4.1/ref/models/fields/#django.db.models.ImageField) so Django can manage them? You would need to sort out how you will be storing the images - probably an S3 bucket for production but a local media volume for dev. You can use django-storages and a different MEDIA_URL configurations for dev and prod.

Reply:
This makes sense.
- Added #91 for discussion